site/content/docs/user/quick-start.md

```diff
@@ -147,6 +147,15 @@ See [Kubernetes imagePullPolicy][Kubernetes imagePullPolicy] for more informatio
 
 See also: [Using kind with Private Registries][Private Registries].
 
+**Note**:
+if you got `Failed to pull image "my-custom-image:unique-tag"` after applying your manifest. You can verify if kind pulled your image into its cluster by calling`crictl images` command inside `kind-control-plane`. 
```

Comment:
we should probably substitute "inside `kind-control-plane`" with "inside one of the nodes" or something along those lines, since it depends on the clustrer name?

Reply:
there's some trade off between correctness and being more confusing for the user just getting started, I'm not actually sure how to best put this one 😞